### ISSUE TITLE
Prevent --tcpip from killing previous instances connected to the same device

### ISSUE THREAD

**Issue by 4nric** [Genymobile/scrcpy#5562]
_Please read the [prerequisites] to run scrcpy._

[prerequisites]: https://github.com/Genymobile/scrcpy#prerequisites

_Also read the [FAQ] and check if your [issue][issues] already exists._

[FAQ]: https://github.com/Genymobile/scrcpy/blob/master/FAQ.md
[issues]: https://github.com/Genymobile/scrcpy/issues

## Environment

 - **OS:** [e.g. Debian, Windows, macOS...] Windows
 - **Scrcpy version:** [e.g. 2.5] 3.0
 - **Installation method:** [e.g. manual build, apt, snap, brew, Windows release...] Windows Release
 - **Device model:** SM-G975F
 - **Android version:** [e.g. 14] 12, 14
## Describe the bug

Since `--tcpip `was added, opening a new terminal and mirroring the same device will close all other instances of the same mirrored device launched with `--tcpip`. Everything else launched with `-s`, `-e`, `-d` are unaffected.

My request would be to allow '--tcpip' be run as many times for a device just like `-s` option.

With `--tcpip`
1. `scrcpy --tcpip=192.168.1.10 --new-display --start-app=?Chrome`
2. `scrcpy --tcpip=192.168.1.10 --new-display --start-app=?Settings`

Both will connect to 192.168.1.10 if adb is not yet connected, then attempt launching specified app.
However, after 1st is opened, the 2nd one also gets opened but, the first one will forcefully close.

With `-s`
1. `scrcpy -s 192.168.1.10 --new-display --start-app=?Chrome`
2. `scrcpy -s 192.168.1.10 --new-display --start-app=?Settings`

Both will fail if adb is not yet connected to `192.168.1.10`
Otherwise, both can multiple windows concurrently.

**Comment by rom1v** [comment]
The reason why `--tcpip` disconnects is that very often, a TCP/IP connection listen in `adb devices` may be "broken" (for example, the device has been rebooted, but adb still list the connection).

The only way to reconnect is to disconnect beforehand.

https://github.com/Genymobile/scrcpy/blob/1d2f16dbb53facdb2ea174578437a2a5afb6aede/app/src/server.c#L840

That being said, we could add an option to not disconnect, for example:

```bash
scrcpy --tcpip=192.168.x.x
# not implemented
scrcpy --tcpip=+192.168.x.x # reconnect
```

**Comment by rom1v** [comment]
I just implemented that (just a few lines of code): 246719f1d6d79d71b4d9cdebf835818868f35d65 (branch `tcpip_reset`)

**Comment by 4nric** [comment]
Thank you! 🥰 This will be useful for --start-app shortcuts
